README.md

```diff
@@ -56,6 +56,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Cixware Pvt. Ltd.](https://cixware.io) | New Baneshwor, Kathmandu | Web and Mobile apps development, DevOps, Cloud services integration and automation
 - [Classic Tech Pvt. Ltd.](https://www.classic.com.np/) | New Baneshwor, Kathmandu | Internet Service Provider (ISP) and Network Service Provider (NSP)
 - [CloudFactory](https://www.cloudfactory.com/) | Bhaisepati, Lalitpur | On-Demand Workforce for Data Work
+- [Code Kavya](https://codekavya.com/) | Pokhara | Codekavya is a design and development company specialized in creating beautiful and scalable full-stack web and mobile applications
```

Comment:
@Parajulibkrm Thank you for the contribution.

Can you also add a specific location so that it follows convention. For example: `Lake Side, Pokhara`.

Reply:
@mesaugat  I have added the specific location. Should be good now.